app/helpers/article_helper.rb

```diff
@@ -11,6 +11,10 @@ def article_restricted?(document)
     document['eds_title'] =~ /^This title is unavailable for guests, please login to see more information./
   end
 
+  def html_fulltext_available?(document)
+    !document['eds_html_fulltext'].blank?
```

Comment:
What do you think about this a method on the `SolrDocument` class instead of a helper that gets passed an instance of a document?